spec/rspec/mocks/partial_double_spec.rb

```diff
@@ -379,6 +390,22 @@ def defined_private_method
         object.send(:defined_private_method)
       end
 
+      it 'runs the before_verifying_double callbacks on any_instance' do
+        expect { |probe|
+          RSpec.configuration.mock_with(:rspec) do |config|
+            config.before_verifying_doubles(&probe)
+          end
+
+          expect_any_instance_of(klass).to receive(:implemented)
+          object.implemented
+        }.to yield_successive_args(
+          *(klass.ancestors + [klass]).map do |target|
+            have_attributes(:target => target)
+          end
+        )
+      end
+
+
```

Comment:
These specs show that the callback is called but don't show _when_ it gets called.  Is it called when `expect_any_instance_of(...).to receive` is used?  Is it called before `implemented` gets called?  Or after those?  It would be nice if the spec would make that explicit (which may require you to move away from using `yield_successive_args`).